lib/react_client.dart

```diff
@@ -245,10 +235,7 @@ final ReactDartInteropStatics _dartInteropStatics = (() {
   var zone = Zone.current;
 
   /// Wrapper for [Component.getInitialState].
-  Component initComponent(
-          ReactComponent jsThis,
-          ReactDartComponentInternal internal,
-          InteropContextValue context,
+  Component initComponent(ReactComponent jsThis, ReactDartComponentInternal internal, InteropContextValue context,
```

Comment:
bah. lol i guess i like it inline for 2-3 or less arguments with a type that is less than 6 characters long each only hahah

Reply:
No wait, i know what it is, I feel like if all the arguments cant fit on 1 line cleanly they should all by stacked... thats it... if only we had the option to change dart fmt.... one can dream